pkg/customAnalyzer/customAnalyzer.go

```diff
@@ -0,0 +1,39 @@
+package customanalyzer
```

Comment:
Can we name this package and directory `custom_analyzer` ?

Reply:
I'm not a big fan of snake case but make sense